CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Version 3.5.0
+-------------
+* Add ``Message-ID`` to the Email models and use it when sending emails. It replaces the
```

Comment:
Do you mind removing the part where this removed ID column in Django's admin?

Reply:
Sorry, I don't understand what you mean by that.

Reply:
Ok. Changed that.